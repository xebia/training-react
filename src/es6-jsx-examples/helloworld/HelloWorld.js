import React from 'react';

const HelloWorld = () => <div>
  <h1>Hello World!</h1>
  <p>this is a paragraph</p>
  <p>Look at this list:</p>
  <ol>
    {[1, 2, 3].map(number => <li key={number}>item {number}</li>)}
  </ol>
</div>;

export default HelloWorld;
