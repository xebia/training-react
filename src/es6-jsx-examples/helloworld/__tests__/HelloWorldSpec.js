import React from 'react';
import renderer from 'react-test-renderer';
import HelloWorld from '../HelloWorld.js';

describe('Component: HelloWorld', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <HelloWorld />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});