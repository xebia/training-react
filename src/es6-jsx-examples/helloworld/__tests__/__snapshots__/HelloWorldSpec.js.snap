// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: HelloWorld should render correctly 1`] = `
<div>
  <h1>
    Hello World!
  </h1>
  <p>
    this is a paragraph
  </p>
  <p>
    Look at this list:
  </p>
  <ol>
    <li>
      item 
      1
    </li>
    <li>
      item 
      2
    </li>
    <li>
      item 
      3
    </li>
  </ol>
</div>
`;
